.gitignore

```diff
@@ -1,3 +1,6 @@
+.*
+!/.gitignore
+!/.travis.yml
```

Comment:
Ignoring all hidden files by default seems like a bit too much 😅 

You can ignore files locally by editing: `.git/info/exclude`.

Reply:
All hidden files except `.gitignore` and `.travis.yml` 😉
But yeah maybe it's better to use `.git/info/exclude` for that 🙂